bundles/Billing/Service/BillingService.php

```diff
@@ -188,6 +188,8 @@ private function getInvoiceArray(Invoice $invoice, bool $withAccount = false)
             'created_by' => $invoice->getCreatedBy(),
             'defaultPayToAccount' => $invoice->getDefaultPayToAccount(),
             'defaultMaterialNumber' => $invoice->getDefaultMaterialNumber(),
+            'periodFrom' => $invoice->getPeriodFrom() ? $invoice->getPeriodFrom()->format('U') : null,
+            'periodTo' => $invoice->getPeriodTo() ? $invoice->getPeriodTo()->format('U') : null,
```

Comment:
For consistency with `exportedDate` you could use `->format('c')` here and use `new Date(…).getTime()` (or `moment`) to convert the date value in https://github.com/itk-dev/jira_economics/pull/109/files#diff-feb28d681f25eea707a612b967b33048192c4c722fcc8f2512014cd2af263b38R88-R89.